source/extensions/filters/http/cache/cache_headers_utils.cc

```diff
@@ -289,52 +305,42 @@ constexpr absl::string_view header_separator = "\n";
 constexpr absl::string_view in_value_separator = "\r";
```

Comment:
this is pre-existing but can you change these to HeaderSeparator and InValueSeparator per convention in https://github.com/envoyproxy/envoy/blob/main/STYLE.md